.github/workflows/cancel.yml

```diff
@@ -12,5 +12,5 @@ jobs:
         with:
           # Ids to cancel core/mpi/gpu/examples/tutorials
           # https://api.github.com/repos/devitocodes/devito/actions/workflows
-          workflow_id: 167582, 203470, 203471, 434862, 501517
+          workflow_id: 167582, 203470, 203471, 434862, 501517, 50473775
```

Comment:
Ok - school me... what does this do?

Reply:
Oh this cancel the workflows currently running when you push again so we don't have thousands of concurrent runs from different commits on the same branch

Reply:
In the url line 14